avocado/core/__init__.py

```diff
@@ -165,7 +166,28 @@ def register_core_options():
                                     help_msg=help_msg)
 
 
+def initialize_plugin_infrastructure():
+    help_msg = 'Plugins that will not be loaded and executed'
+    future_settings.register_option(section='plugins',
+                                    key='disable',
+                                    key_type=list,
+                                    default=[],
+                                    help_msg=help_msg)
+
+    kinds = pkg_resources.get_entry_map('avocado-framework').keys()
+    plugin_types = [kind[8:] for kind in kinds
+                    if kind.startswith('avocado.plugins.')]
+    for plugin_type in plugin_types:
+        help_msg = 'Execution order for "%s" plugins' % plugin_type
+        future_settings.register_option(section=plugin_type,
```

Comment:
Any specific reason to not use the same section "plugins" ?

Reply:
> Hi @clebergnu, any reason to not use the same section "plugins" ?

I don't understand your question... the sections are named `plugins.$type` for the `order` key. They have not changed.

Reply:
Sorry, never mind. My bad.